api/types/platform_helpers.go

```diff
@@ -25,6 +25,7 @@ func OCIPlatformToProto(platforms []oci.Platform) []*Platform {
 	for i := range platforms {
 		ap[i] = &Platform{
 			OS:           platforms[i].OS,
+			OsVersion: 	  platforms[i].OSVersion,
```

Comment:
I think we should have this be `OSVersion` but am not sure what tooling is in the repo to do this now that gogo extensions.